packages/flutter/lib/src/widgets/image.dart

```diff
@@ -1185,6 +1185,8 @@ class _ImageState extends State<Image> with WidgetsBindingObserver {
     setState(() {
       _imageInfo = imageInfo;
       _loadingProgress = null;
+      _lastException = null;
+      _lastStack = null;
```

Comment:
Do we need to do this in `_handleImageChunk` as well?

Reply:
Yes - we should add the same after line 1198.

Reply:
Yes we do.